pandas/tests/indexes/period/test_period.py

```diff
@@ -561,6 +561,12 @@ def test_insert(self):
             result = period_range('2017Q1', periods=4, freq='Q').insert(1, na)
             tm.assert_index_equal(result, expected)
 
+    def test_comp_op(self):
```

Comment:
Slight nitpick: this test name isn't that descriptive of what it's testing. I'd prefer something like `test_comparing_different_lengths_raises`.